cypress/integration/pages/details.spec.js

```diff
@@ -2,7 +2,7 @@
 describe('Details Page', () => {
   beforeEach(() => {
     cy.setHackneyCookie(true);
-    cy.visit('http://localhost:3001/customers/5/view');
+    cy.visit('http://localhost:3001/customers/5/view#shared_plan');
```

Comment:
Could we just set this cookie, similar to how we do for the Hackney token cookie? Then we are set up for the future where there may be multiple combinations of flags we need to test - and we don't need to have a whole bunch of `cy.visit`s running before each test.

I wouldn't want this to not be tested though, so maybe an integration test for this URL to check it does get set too?

Reply:
Fixed it up a bit now